src/soilbiogeochem/SoilBiogeochemDecompCascadeBGCMod.F90

```diff
@@ -408,8 +408,8 @@ subroutine init_decompcascade_bgc(bounds, soilbiogeochem_state_inst, soilstate_i
       i_pro_som = i_lig_lit + 1
       floating_cn_ratio_decomp_pools(i_pro_som) = .false.
       decomp_cascade_con%decomp_pool_name_restart(i_pro_som) = 'soil1'
-      decomp_cascade_con%decomp_pool_name_history(i_pro_som) = 'SOIL1'
-      decomp_cascade_con%decomp_pool_name_long(i_pro_som) = 'soil 1'
+      decomp_cascade_con%decomp_pool_name_history(i_pro_som) = 'PAS_SOM'
+      decomp_cascade_con%decomp_pool_name_long(i_pro_som) = 'passive soil organic matter'
```

Comment:
@wwieder let's make sure that I labeled the three SOMs correctly. In the first refactor I had chosen other names for the corresponding indices:
- i_pro_som for "protected"
- i_rec_som for "recalcitrant"
- i_avl_som for "available"
If I have now correctly mapped "protected" to "passive", "recalcitrant" to "slow", and "available" to "active", then I think I should also rename the indices for consistency to i_pas_som, i_rec_som, and i_act_som. Let me know if you disagree.

Reply:
This is likely fine, but we should add carbon and nitrogen to both `name_history` and `name_long` (as is done with the BGC / CENTURY pools).

Reply:
> This is likely fine, but we should add carbon and nitrogen to both `name_history` and `name_long` (as is done with the BGC / CENTURY pools).

The C/carbon and N/nitrogen labels get added to the history fields when the variables are written to history. This way the section of the code that we are looking at here can be written once for both C and N.